File: Pyvvo.Logistics.Model/Model/Parameter.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Pyvvo.Logistics.Model
{
    public class Parameter
    {
        [Key, Required] public long Id { get; set; }
        [Required] public long WeightUnitId { get; set; }
        [Required] public long DimensionUnitId { get; set; }
        [Required] public long CreatedById { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public bool IsActive { get; set; }
        public string Name { get; set; }
        public double Length { get; set; }
        public double Width { get; set; }
        public double Height { get; set; }
        public double Weight { get; set; }
        public MeasurementUnit WeightUnit { get; set; }
        public MeasurementUnit DimensionUnit { get; set; }
        public User CreatedBy { get; set; }
    }
}

File: Pyvvo.Logistics/ClientApp/src/app/service/Model/parameter.ts
import { User } from "./user";
import { MeasurementUnit } from "./measurement-unit";

export class Parameter {
  id: number;
  createdOn: Date;
  createdOnLocalTime: string;
  updatedOn: Date;
  updateOnLocalTime: string;
  name: string;
  isActive: boolean;
  length: number;
  width: number;
  height: number;
  weight: number;
  weightUnit: MeasurementUnit;
  dimensionUnit: MeasurementUnit;
  createdBy: User;
}
